### PR TITLE
Add some production metrics

### PR DESCRIPTION
**Pull request by djeebus**

### CODE REVIEW

File: aggregate.go [hunk 1]
```diff
@@ -78,6 +78,13 @@ func (ao *aggregateOptions) formatIgnoredLabels() {
 	sort.Strings(ao.ignoredLabels)
 }
 
+func (a *aggregate) Len() int {
+	a.familiesLock.RLock()
+	count := len(a.families)
```
Comment (mplachter) [inline aggregate.go:83]: This will only count the metric with a name not unique names with the labels

**Review by mplachter — APPROVED** [review]
🚀

File: aggregate.go (hunk 1 above)
Comment (djeebus) [reply aggregate.go:83]: yeah, that's what I was looking for here. total number of unique names, not unique name+label combinations. might be worth adding that metric in as well though, right? i'll add that

Comment (djeebus) [reply aggregate.go:83]: ... will add on a follow up

**Pull request merged**